tutorials/vue/vue-d3.md

```diff
@@ -0,0 +1,151 @@
+[D3.js](https://d3js.org/) is a great library for visualizing data and displaying it in your projects.
+This library gives you the building blocks to make cool charts in projects.
+
+Here is an example of using `D3.js` in Vue2 to create a bar graph.
+Below is the code:
+
+```html
+<script src="https://d3js.org/d3.v6.js"></script>
+<script src="https://unpkg.com/vue"></script>
+<div class = "app">{{message}}
+<bargraph></bargraph>
+</div>
+<script type = "text/javascript">
+Vue.component('bargraph', {
+    mounted(){
+        this.retrieveGraph();
+    },
+    methods: {
+        retrieveGraph: function(){
+            let margin = ({top: 30, right: 30, bottom: 70, left: 60});
+            let width = 460 - margin.left - margin.right;
+            let height = 400 - margin.top - margin.bottom;
+            let info = [{Country: "United States", Value: "12394"},
```

Comment:
Pull `info` out of the component and make it global to make this a bit more readable